src/Modules/OrchardCore.Commerce/Extensions/HttpContextExtensions.cs

```diff
@@ -12,6 +12,9 @@ public static class HttpContextExtensions
     public static Task<UserAddressesPart> GetUserAddressAsync(this HttpContext context) =>
         context.RequestServices.GetRequiredService<UserManager<IUser>>().GetUserAddressAsync(context.User);
 
+    public static Task<UserDetailsPart> GetUserDetailsAsync(this HttpContext context) =>
+        context.RequestServices.GetRequiredService<UserManager<IUser>>().GetUserDetailsAsync(context.User);
```

Comment:
Would be nicer to break down this line before every `.`
Also there are many elements to this chain of calls and I can't see any null checks or possible breaks in the call.
For example it is possible for `.GetRequiredService` to throw `InvalidOperationException`

Reply:
IMO it's short enough to stay on one line.

I think `UserManager` is pretty much guaranteed to be found in all use cases of these extension methods, so it strikes me as unnecessary to add further null-checking here. This also seems to be in line with how `GetRequiredService()` is used in other solutions like OSOCE.

Reply:
I just don't like stacking calls after eachother no matter how long they are to be honest.
We can have chain without any further checks if there are more examples of this.

Reply:
For example, it's used similarly in [Helpful-Libraries/EnvironmentHttpContextExtensions](https://github.com/Lombiq/Helpful-Libraries/blob/1687c7b510ada082ea39c8f91451639b658ed552/Lombiq.HelpfulLibraries.AspNetCore/Extensions/EnvironmentHttpContextExtensions.cs#L17).

If you mean the following styling, IMO it uses too much vertical space and not enough horizontal. As far as I have seen, we most commonly put these on one line as long as they don't get too long.


```
context
    .RequestServices
    .GetRequiredService<UserManager<IUser>>()
    .GetUserDetailsAsync(context.User);
```

Reply:
Yes I was thinking about the above styling.
Yes, as you say it doesn't interfere with our 120-150 char length rule but breaking down the lines in my opinion makes the chain so much more readable. 
But this is "just" styling and one line so I consider this case resolved by leaving it as 1liner.